src/Console/CommandRunner.php

```diff
@@ -41,6 +42,13 @@ class CommandRunner
     protected $root;
 
     /**
+     * Alias mappings.
```

Comment:
It might help to understand how the mapping should be set, if we show an example like https://github.com/cakephp/cakephp/pull/10819/files#diff-5746a5b6845a932f92018ff514ae2000R62